### ISSUE TITLE
blender jobs with linked cached cloth-sim objects

### ISSUE THREAD

**Issue by aceman500** [jamesyonan/brenda#34]
I have a blend scene with linked objects with cloth sim on them. The sim is done in the main scene file and the blend-cache is saved in the main scene file directioy .  I packed the libraries tared up the blend file and the cache directory and submitted it to a work queue.
Now as we all know, for whatever stupid reason, blender likes to delete thees kind of cache files next time the scene is opened. The usual workaround is to set proper permissions on the cache directory. However brenda runs blender as root on the instances and the permissions are meaningless, so blender deletes the cache files and breaks the cloth sim. Cloth sim works fine on local sims.. Blender does not wish to delete thees cache files..
Any ideas for a workaround?

**Comment by tuttlen** [comment]
If I am not mistaken brenda is a shell script? At least that is the one I
used. If this is the case couldn't you modify permissions on the shell
executable or add to a group that doesn't have permission to delete but has
full control of that directory. Another way would be to alter the shell
script updating or jailing the blender process.

**Comment by aceman500** [comment]
Yes, brenda is a collection of python scripts. I don't know yet how they function and my python is not so good, so I thought here would be a good place to ask. In addition I assume there is a reason it's run as root. Otherwise brenda is real cool and it runs flawlessly, The problem im having is not a brenda problem at all rather a problem with the way blender handles cached file linking which has been a known issue for quit a while now. My workaround is to just change the permissions so blender can't delete them and it works fine. But in the case of using brenda to kick off the jobs, that trick no longer works.

**Comment by aceman500** [comment]
So at anyrate, I guess my basic question is: Can I run blender and the brenda scripts as a non-root user.
I have a nice scene and I finally get to user the power of AWS and this annoying issue cropped up.
Thanks for you help.

**Comment by jldonovan3** [comment]
Has someone figured out a workaround or a solution to this? I think I'm having a similar problem trying to render a scene with a smoke sim. I have my directory with my .vdb cache that's referenced in the .blend file, and I get this:

```
loadTile: No noise tile '/tmp/blender_rt5jh0k/noise.wavelets' found.
Generating new 3d noise tile size=128^3 
saveTile: Noise tile file '/tmp/blender_rt5jh0k/noise.wavelets' saved.
Generating new 3d noise done
Noise tile file '/tmp/blender_ET1Ps2/noise.wavelets' loaded.
```

Then nothing. The process eventually gets killed. Googling turned up a bunch of old threads on the Blender dev bug reporting pages, but nothing that helps my situation.
